<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KTimerBox</class>
 <widget class="QWidget" name="KTimerBox">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>416</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>KTimerBox</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="5,1,1,1">
     <item>
      <widget class="QCalendarWidget" name="calendarWidget">
       <property name="font">
        <font>
         <family>仿宋</family>
         <pointsize>16</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSetTime">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="hLayoutTime" stretch="1,2,3">
          <item>
           <widget class="QLabel" name="label_time">
            <property name="text">
             <string>提醒时间</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTimeEdit" name="timeEditSetTime">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxSetAgain">
       <property name="title">
        <string/>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,2,3">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>25</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">border-image: url(:/KTodoMainWindow/images/setAgain.png);</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxSetAgain">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">border:none;</string>
            </property>
            <item>
             <property name="text">
              <string>设置重复</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>每天重复</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>每周重复</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>每月重复</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>不重复</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>15</number>
       </property>
       <property name="topMargin">
        <number>9</number>
       </property>
       <property name="rightMargin">
        <number>15</number>
       </property>
       <property name="bottomMargin">
        <number>9</number>
       </property>
       <item>
        <widget class="QPushButton" name="pbnCancel">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">border:2px groove gray;
border-radius:5px;
padding:2px 4px;
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pbnOK">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="styleSheet">
          <string notr="true">
color: rgb(255, 255, 255);
background-color: rgb(62, 193, 135);
border:2px groove gray;
border-radius:5px;
padding:2px 4px;</string>
         </property>
         <property name="text">
          <string>确定</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ktodomainwindow.qrc"/>
 </resources>
 <connections/>
</ui>
